inc/Engine/CriticalPath/CriticalCSSSubscriber.php

```diff
@@ -764,4 +776,73 @@ public function stop_critical_css_generation() {
 		delete_transient( 'rocket_critical_css_generation_process_running' );
 		delete_transient( 'rocket_critical_css_generation_process_complete' );
 	}
+
+	/**
+	 * Display a notice to pass from CPCSS to RUCSS.
+	 *
+	 * @return void
+	 */
+	public function switch_to_rucss_notice() {
+		$boxes = get_user_meta( get_current_user_id(), 'rocket_boxes', true );
+
+		if ( in_array( __FUNCTION__, (array) $boxes, true ) ) {
+			return;
+		}
+
+		if ( ! $this->options->get( 'async_css', 0 ) ) {
+			return;
+		}
+
+		if ( $this->user->is_license_expired() ) {
+			return;
+		}
+
+		if ( isset( $screen->id ) && 'settings_page_wprocket' !== $screen->id ) {
+			return;
+		}
+
+		rocket_notice_html(
+			[
+				'status'                 => 'wpr-warning',
+				'dismissible'            => '',
+				'dismiss_button'         => 'test',
+				'message'                => sprintf(
+					// translators: %s = opening bold tag, %s = closing bold tag.
+					__( 'We highly recommend the %1$s updated Remove Unused CSS %2$s for a better CSS optimization. Load CSS Asynchronously is always available as a back-up.', 'rocket' ),
+					'<b>',
+							'</b>'
+				),
+				'action'                 => 'switch_to_rucss',
+				'dismiss_button_message' => __( 'Stay with old option', 'rocket' ),
+			]
+		);
+
+	}
+
+	/**
+	 * Switch to RUCSS.
+	 *
+	 * @return void
+	 */
+	public function switch_to_rucss() {
+		if ( ! isset( $_GET['_wpnonce'] ) || ! wp_verify_nonce( sanitize_key( $_GET['_wpnonce'] ), 'rucss_switch' ) ) {
+			wp_nonce_ays( '' );
+		}
```

Comment:
You can use https://developer.wordpress.org/reference/functions/check_admin_referer/ instead